components/DayRow.tsx

```diff
@@ -0,0 +1,71 @@
+import { Text, TouchableOpacity, StyleSheet, useColorScheme, TextInput } from 'react-native'
+import React from 'react'
+import Colors from '../constants/Colors';
+
+export default function DayRow({ WeekDay, index, text, editableDay, handleDayClick, handleTextChange }: { WeekDay: string, index: number, text: string | undefined, editableDay: number | null, handleDayClick: (day: number) => void, handleTextChange: (text: string) => void }) {
+
+  const colorScheme = useColorScheme() ?? 'light';  
+  return (
+    <TouchableOpacity
+            key={index}
+            onPress={() => handleDayClick(index)}
+            style={[
+              styles.dayContainer, 
+              {backgroundColor: index % 2 == 0 ? Colors[colorScheme].listBackgroundColor1 : Colors[colorScheme].listBackgroundColor2},
+            ]}
+          >
+            <Text style={[styles.weekdaysText, { fontWeight: 'bold' }]}>{WeekDay}</Text>
```

Comment:
Maybe add `color: Colors[colorScheme].text` here and below, so the text isn't black on gray in dark mode